pkg/pillar/cmd/volumemgr/create.go

```diff
@@ -74,8 +73,7 @@ func createContainerVolume(ctx *volumemgrContext, status types.VolumeStatus,
 	srcLocation string) (bool, string, error) {
 
 	created := false
-	filelocation := fmt.Sprintf("%s/%s#%d", status.VolumeDir,
-		status.VolumeID.String(), status.GenerationCounter)
+	filelocation := status.VolumeName()
 	if err := containerd.SnapshotPrepare(filelocation, srcLocation); err != nil {
 		log.Errorf("Failed to create ctr bundle. Error %s", err)
```

Comment:
You check for existing VM image. Why don't you need to or want to do that for an OCI volume?